docs/tutorials/Uniquery.md

```diff
@@ -1,5 +1,6 @@
 **Uniquery Documentation V1**
 
+
 KodaDot now has its’ own graphQL builder for our Subquery and SubSquid!
```

Comment:
- please use `'` instead of `’`
- graphQL -> GraphQL [Brand Guid](https://graphql.org/brand)
- we removed support for `Subquery` since kodadot/nft-gallery#3640